jwt-react/src/components/Login/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useContext } from 'react';
+import { UserContext } from '../../contexts/userContext.js'
+import { loginUser, logoutUser } from '../../api/auth.js'
+
+
+
+function Login() {
+  const [username, setUsername] = useState("")
+  const [password, setPassword] = useState("")
+  const [errorDisplay, setErrorDisplay] = useState("")
+  const {user, setUser, isUserLoggedIn} = useContext(UserContext)
+
+  const onLoginFormSubmit = (event) => {
+    event.preventDefault();
+    if (!isValidForm()){
+      return
+    }
+    loginUser(username, password).then((data)=>{
+      setUser({username: username})
+    }).catch((error)=> {
+      setErrorDisplay()
+    })
+  }
+
+  const isValidForm = () => {
+    setErrorDisplay("")
+    if (username === "" || password === ""){
+      setErrorDisplay("username and password can't be empty. try user: test, password: test")
+      return false;
+    }
+    return true;
+  }
+
+  const logout = () => {
+    logoutUser()
+    setUser(null);
+  }
+
+  return <div>
+    {
+      isUserLoggedIn() ?
+      <React.Fragment>
+      <h3>{user.username} is currently Logged in</h3>
```

Comment:
Hey thanks for the review on this, I wasn't sure really the scope of this. 

I didn't add the `react-router-dom` (https://www.npmjs.com/package/react-router-dom) because I thought it would fall out of scope. Looking as this idea was referenced a couple of times I will modify this accordingly and add a couple of pages.